sdk/communication/AzureCommunicationCalling/AzureCommunicationCalling-Swift.h

```diff
@@ -0,0 +1,1027 @@
+// ACSCallingShared
+// This file was auto-generated from ACSCallingModel.cs.
+
+import Foundation
+
+import AzureCommunication
+//#import <AzureCore/AzureCore-Swift.h>
+
+// Enumerations.
+/// Additional failed states for Azure Communication Services
+public struct CallingCommunicationErrors : OptionSet {
+
+    public init(rawValue: Int)
+
+    
+    /// No Audio permissions available.
+    public static var noAudioPermission: CallingCommunicationErrors { get }
+
+    /// No Video permissions available.
+    public static var noVideoPermission: CallingCommunicationErrors { get }
+
+    /// No Video and Audio permissions available.
+    public static var noAudioAndVideoPermission: CallingCommunicationErrors { get }
+
+    /// Failed to process push notification payload.
+    public static var receivedInvalidPushNotificationPayload: CallingCommunicationErrors { get }
+
+    /// Recieved empty/invalid notification payload.
+    public static var failedToProcessPushNotificationPayload: CallingCommunicationErrors { get }
+
+    /// Recieved invalid group Id.
+    public static var invalidGuidGroupId: CallingCommunicationErrors { get }
+
+    /// Push notification device registration token is invalid.
+    public static var invalidPushNotificationDeviceRegistrationToken: CallingCommunicationErrors { get }
+
+    /// Cannot create multiple renderers for same device or stream.
+    public static var multipleRenderersNotSupported: CallingCommunicationErrors { get }
+
+    /// Renderer doesn't support creating multiple views.
+    public static var multipleViewsNotSupported: CallingCommunicationErrors { get }
+}
+
+/// Direction of the camera
+public enum CameraFacing : Int {
+
+    
+    /// Unknown
+    case unknown = 0
+
+    /// External device
+    case external = 1
+
+    /// Front camera
+    case front = 2
+
+    /// Back camera
+    case back = 3
+
+    /// Panoramic camera
+    case panoramic = 4
+
+    /// Left front camera
+    case leftFront = 5
+
+    /// Right front camera
+    case rightFront = 6
+}
+
+/// Describes the video device type
+public enum VideoDeviceType : Int {
+
+    
+    /// Unknown type of video device
+    case unknown = 0
+
+    /// USB Camera Video Device
+    case usbCamera = 1
+
+    /// Capture Adapter Video Device
+    case captureAdapter = 2
+
+    /// Virtual Video Device
+    case virtual = 3
+}
+
+/// Local and Remote Video Stream types
+public enum MediaStreamType : Int {
+
+    
+    /// Video
+    case video = 0
+
+    /// Screen share
+    case screenSharing = 1
```

Comment:
Seems like most of the enum consistency issues are fixed except this. 1 and 2?

Reply:
yes those will be fixed in another PR, here we are only focussing on. the GA blockers.